src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/Tkn.java

```diff
@@ -142,4 +142,14 @@
      */
     void deleteResource(String namespace, String resource) throws IOException;
 
+    /**
+     * Start the execution of a task
+     *
+     * @param namespace the namespace of the task
+     * @param task the task that has to be run
+     * @param args args to start task (contains input/output params/resources)
+     * @throws IOException if communication errored
+     */
+    void runTask(String namespace, String task, String args) throws IOException;
```

Comment:
I don't like args being passed as a single parameter. You will face issues when parameters have space in there values. Should be an an array or a collection (list/set)